io.github.Fontforge.appdata.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Joe Da Silva <digital@joescat.com> -->
+<application>
```

Comment:
You should run `appstream-util upgrade` on this file.

Reply:
done